sigstore-testkit/src/main/kotlin/dev/sigstore/testkit/BaseGradleTest.kt

```diff
@@ -38,17 +39,44 @@ open class BaseGradleTest {
     companion object {
         val isCI = System.getenv().containsKey("CI") || System.getProperties().containsKey("CI")
 
+        val EXTRA_LOCAL_REPOS  = System.getProperty("sigstore.test.local.maven.repo").split(File.pathSeparatorChar)
+
+        val SIGSTORE_JAVA_CURRENT_VERSION =
+            TestedSigstoreJava.Version(
+                System.getProperty("sigstore.test.current.version")
+            )
+
         @JvmStatic
         fun gradleVersionAndSettings(): Iterable<Arguments> {
             if (!isCI) {
                 // Make the test faster, and skip extra tests with the configuration cache to reduce OIDC flows
                 // Gradle 7.2 fails with "No service of type ObjectFactory available in default services"
-                return listOf(arguments("7.3", ConfigurationCache.OFF))
+                return listOf(arguments(TestedGradle("7.3", ConfigurationCache.ON)))
             }
             return mutableListOf<Arguments>().apply {
-                add(arguments("7.3", ConfigurationCache.ON))
-                add(arguments("7.5.1", ConfigurationCache.ON))
-                add(arguments("7.5.1", ConfigurationCache.OFF))
+                add(arguments(TestedGradle("7.3", ConfigurationCache.ON)))
+                add(arguments(TestedGradle("7.5.1", ConfigurationCache.ON)))
+                add(arguments(TestedGradle("7.5.1", ConfigurationCache.OFF)))
+            }
+        }
+
+        @JvmStatic
+        fun sigstoreJavaVersions(): Iterable<Arguments> {
+            return mutableListOf<Arguments>().apply {
+                add(arguments(SIGSTORE_JAVA_CURRENT_VERSION))
+                if (isCI) {
+                    add(arguments(TestedSigstoreJava.Default))
+                    add(arguments(TestedSigstoreJava.Version("0.2.0")))
```

Comment:
Is using "0.2.0" going to be hard to track?

Reply:
Any suggestions here?

Reply:
hrmm not really. maybe leave this like this for now?

Reply:
The PR that adds bundle removes 0.2.0 from here.
I suggest we just keep it as is to see how it works before we move it to properties or something else.